website/blog/2021/06/21/use-Java-to-write-Apache-APISIX-plugins.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "How to Write an Apache APISIX Plugin in Java"
-slug: 2021/06/21/use-java-to-write-apache-apisix-plugins
+title: "How to use Java to write plugins for API Gateway?"
+slug: 2021/06/21/how-to-use-java-to-write-plugins-for-api-gateway
```

Comment:
Please note, once you updated the slug, the article's original URL will be 404.

Reply:
Got it. But we need a better uri.

Reply:
> the article's original URL will be 404.

Can we use https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects/ ？

Reply:
Previously tried plugin-client-redirect, but it was not able to pass checks.

https://github.com/apache/apisix-website/pull/903#issuecomment-1050774757

Reply:
> Previously tried plugin-client-redirect, but it was not able to pass checks.
> 
> [#903 (comment)](https://github.com/apache/apisix-website/pull/903#issuecomment-1050774757)

What a strange problem.🤔